client/interfaces.ts

```diff
@@ -1197,7 +1197,7 @@ export interface IEventsPlanningContentPanelProps {
 
 export interface IPlanningExportTemplate extends IBaseRestApiResponse {
     name: string;
-    type: 'event' | 'planning' | 'combined';
+    type: 'events' | 'planning' | 'combined';
```

Comment:
Does this cause lint error? As the `planning_export_template` schema registered with Eve is actually `event`:
https://github.com/superdesk/superdesk-planning/blob/develop/server/planning/planning_export_templates.py#L29

Reply:
yes, it was causing a lint error, that's why I changed it.

Reply:
I chose the incorrect type name for the `events_planning_filters` resource. We'll have to tackle this with another ticket (along with a migration script to fix it). This should be okay for now